src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -43,13 +43,14 @@ internal partial class CurlHandler : HttpMessageHandler
 
         private volatile bool _anyOperationStarted;
         private volatile bool _disposed;
-        private bool _automaticRedirection = true;
         private IWebProxy _proxy = null;
         private ICredentials _serverCredentials = null;
         private ProxyUsePolicy _proxyPolicy = ProxyUsePolicy.UseDefaultProxy;
         private DecompressionMethods _automaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;
         private SafeCurlMultiHandle _multiHandle;
         private GCHandle _multiHandlePtr = new GCHandle();
+        private bool _automaticRedirection = true;
+        private int _maxAutomaticRedirections = 50;
```

Comment:
@stephentoub Yes, one could create a shared source file in src\Common\src\System\Net\Http\... that defined an internal static class (HttpHandlerDefaults) with those constants defined as public fields.  Then that file would be used in the multiple .CSPROJ files to bring in the values.